lib/wasix/src/runners/mod.rs

```diff
@@ -23,7 +23,10 @@ use crate::runtime::{
 /// instance (the "guest").
 #[derive(Debug, Clone, PartialEq, Eq, serde::Serialize, serde::Deserialize)]
 pub struct MappedDirectory {
+    /// The absolute path for a directory on the host filesystem.
```

Comment:
Should we have a constructor `MappedDirectory::new()` that ensures the struct only holds absolute paths? And make the fields private?

(not required for merge)

Reply:
Yeah, I was thinking about that as well. 

It'd be nice to enforce correctness through a constructor, but at the same time it's really nice to be able to destructure with `let MappedDir { host, guest } = ...` instead of needing to write `mapped_dir.host` and `mapped_dir.guest` all over the place.